ext/mysqli/mysqli.stub.php

```diff
@@ -120,7 +120,7 @@ public function init() {}
      * @return bool
      * @alias mysqli_kill
      */
-    public function kill(int $connection_id) {}
+    public function kill(int $process_id) {}
```

Comment:
```suggestion
    public function kill(int $thread_id) {}
```
Given that the corresponding getter is mysql_thread_id(), possibly use that term? I'll admit that the used terminology here is really inconsistent though, so I'm also fine with `$process_id`.